R/curve_interval.R

```diff
@@ -189,7 +188,7 @@ curve_interval = function(.data, ..., .along = NULL, .width = .5,
 
     # if the values we are going to summarise are not already list columns, make them into list columns
     # (making them list columns first is faster than anything else I've tried)
-    if (!all(map_lgl(data[,names(col_exprs)], is.list))) {
+    if (!all(as.logical(unlist(lapply(data[,names(col_exprs)], is.list))))) {
```

Comment:
can we define a `map_lgl_()`  patterned after `map_dbl_()` (previously `vapply_dbl()`) in `util.R`? Using a definition like that (based on `vapply()`) will be faster and safer than unlist + lapply.

Reply:
k